### PR TITLE
Change the infrastucture of the docs to jupyterbook

### PR DESCRIPTION
**Pull request by lschwetlick**
I am starting to fix the user guides section 
- [x] Experiment Types
- [x] index
- [ ] interval bias
- [ ] options-dictionary
- [ ] plot-functions
- [ ] pooling-utility
- [ ] priors
- [ ] result-struct

### CODE REVIEW

**Comment by lschwetlick** [comment]
I might need some help- I left a bunch of TODOs in the options-dictionary file, because I didnt know enough about them. Especially the significance of the moving bounds...

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          auto-activate-base: false
+          miniconda-version: 'latest'
```
Comment (otizonaizit) [inline .github/workflows/build-docs.yml:21]: why are we installing `conda` at all? Further down we use `pip` to install things...
Is this a relic of some other project or is `conda` really needed for something?

File: .github/workflows/build-docs.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          python-version: 3.11
```
Comment (otizonaizit) [inline .github/workflows/build-docs.yml:22]: python 3.13 has been released recently. I think it's better to build the docs with the latest version of Python.

**Comment by otizonaizit** [comment]
@guillermoaguilar : thanks!!! It's exciting to see this thing take form :-)

File: .github/workflows/build-docs.yml (hunk 1 above)
Comment (guillermoaguilar) [reply .github/workflows/build-docs.yml:21]: you're right, this was carried over from another project. Removing...

File: .github/workflows/build-docs.yml (hunk 2 above)
Comment (guillermoaguilar) [reply .github/workflows/build-docs.yml]: changed.

**Pull request merged**